Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -75,12 +75,17 @@ open class MongoSwiftTestCase: XCTestCase {
 
     /// Generates a unique collection name of the format "<Test Suite>_<Test Name>_<suffix>". If no suffix is provided,
     /// the last underscore is omitted.
+    ///
+    /// For compatibility with older servers, this name will be truncated so the entire namespace fits in 120
```

Comment:
some of the newer spec tests resulted in names that were too long, so I added some truncation logic.